src/style.css

```diff
@@ -2,0 +1,96 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(/src/fonts/Avenir.ttc) format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Book', sans-serif;
+  src: url(/src/fonts/Avenir-Book.ttf);
```

Comment:
Consider specifying the format for the font file in the `src` attribute for consistency, as done in other `@font-face` rules.